counter/src/Counter/Counter.jsx

```diff
@@ -1,8 +1,14 @@
 import React from "react"
+import { Row, Col, Button, Alert, Card } from "react-bootstrap"
 
 export const Counter = props =>
-  <div>
-    Count: {props.count}<br/>
-    <button type="button" onClick={props.onInc}>Inc</button>
-    <button type="button" onClick={props.onDec}>Dec</button>
-  </div>
+  <Row>
+    <Col md={2}>
```

Comment:
FYI, this means your column spans 12 below `md` (default) and then suddenly snaps to just 2. If that’s unexpected to you, I plan to give an intro to Bootstrap Grid on Friday 😄

Reply:
The bootstrap docs don't explain how this works exactly, and there doesn't seem to be an "all of them" prop, like `col` we have in `cv-web-core`, so I just copied their examples as well as I could.